test/requirements.txt

```diff
@@ -0,0 +1 @@
+retry
```

Comment:
can this be handled via rosdep via `test_depend`?

Reply:
+1 - I would declare everything as `test_depend`, and let the shell run `rosdep` to install the missing packages?